### PR TITLE
feat: support for document model version 126

### PR DESCRIPTION
**Pull request by jedrichards**
Refs https://github.com/sketch-hq/Sketch/issues/28644

### CODE REVIEW

File: schema/objects/font-data-ref.schema.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+title: Font Data Ref
+description: Defines a reference to base64 font data within the document bundle
+type: object
+properties:
+  _class: { const: MSJSONOriginalDataReference }
+  _ref_class: { const: MSEmbeddedFontData }
```
Comment (christianklotz) [inline schema/objects/font-data-ref.schema.yaml]: Should this be `embeddedFontData` and a replacement in `SketchModel` similar to `embeddedFontReference` or does this not apply to `_ref_class`?

Comment (jnozzi) [reply schema/objects/font-data-ref.schema.yaml]: @christianklotz EmbeddedFontReference has an EmbeddedFontData similar to ImageData.

Comment (jedrichards) [reply schema/objects/font-data-ref.schema.yaml]: I think @jnozzi wanted to keep the implementation of this part as close as possible to image data references, which use `MSImageData`.

Comment (jedrichards) [reply schema/objects/font-data-ref.schema.yaml]: And yeah, looking MSJSONArchiver it doesn't seem as those the replacements are setup to work with `_ref_class` values ...

Comment (christianklotz) [reply schema/objects/font-data-ref.schema.yaml]: 👍

**Pull request merged**